chef-config/lib/chef-config/config.rb

```diff
@@ -235,39 +235,39 @@ def self.derive_path_from_chef_repo_path(child_path)
 
     # Location of data bags on disk. String or array of strings.
     # Defaults to <chef_repo_path>/data_bags.
-    default(:data_bag_path) { derive_path_from_chef_repo_path("data_bags") }
+    default(:data_bag_path) do derive_path_from_chef_repo_path("data_bags") end
 
     # Location of environments on disk. String or array of strings.
     # Defaults to <chef_repo_path>/environments.
-    default(:environment_path) { derive_path_from_chef_repo_path("environments") }
+    default(:environment_path) do derive_path_from_chef_repo_path("environments") end
 
     # Location of groups on disk. String or array of strings.
     # Defaults to <chef_repo_path>/groups.
-    default(:group_path) { derive_path_from_chef_repo_path("groups") }
+    default(:group_path) do derive_path_from_chef_repo_path("groups") end
 
     # Location of nodes on disk. String or array of strings.
     # Defaults to <chef_repo_path>/nodes.
-    default(:node_path) { derive_path_from_chef_repo_path("nodes") }
+    default(:node_path) do derive_path_from_chef_repo_path("nodes") end
```

Comment:
if i'm understanding it correctly this seem incorrectly applied by rubocop?  these functions just do some string wrangling and return a value.

Reply:
`rubocop` doesn't know that. It is just looking at the name / context. Since this isn't being assigned or returned, it looks like it is for side-effect. I'd agree based on this code here.